### DISCUSSION TITLE
How do I connect when I'm not on my home network

### DISCUSSION THREAD

**Discussion by Melsana** [photoprism/photoprism#4539] [General]
So I'm a complete noob and a lot of this is way outside my wheelhouse.  What I want is a way to be able to view my full photo collection from my phone no matter where I am.  I got the software set up and I can see it on my phone when I'm on my home network, but I can't figure out how to access it from outside my home network.  I've tried following the instructions for using a reverse proxy and setting up https... but I'm sure I'm missing some vital piece, because no matter what I try it just says site can't be reached.  Do I need to have my own domain url or can I just use my public ip address?  All the instructions seem to act like I should have my own domain, but frankly I don't really want to pay for a domain just for this, so I'm not sure what to do.  I'm not quite sure what I should be replacing "example.com" with in the examples.

Thanks for any help you can provide.

**Comment by DJKeterson** [comment]
> So I'm a complete noob and a lot of this is way outside my wheelhouse. What I want is a way to be able to view my full photo collection from my phone no matter where I am. I got the software set up and I can see it on my phone when I'm on my home network, but I can't figure out how to access it from outside my home network. I've tried following the instructions for using a reverse proxy and setting up https... but I'm sure I'm missing some vital piece, because no matter what I try it just says site can't be reached. Do I need to have my own domain url or can I just use my public ip address? All the instructions seem to act like I should have my own domain, but frankly I don't really want to pay for a domain just for this, so I'm not sure what to do. I'm not quite sure what I should be replacing "example.com" with in the examples.
> 
> Thanks for any help you can provide.

@Melsana you're right, there is a piece of the puzzle missing. You should port forward the service on your home router to the local machine which is running PhotoPrism and that will allow you to connect from anywhere using your current public IP address. Example being:
http://188.252.137.111:2342 - if you left the default port in use.

I guess you know how to access the router, usually it is found by "ipconfig" command and listed under line "Default Gateway : 192.168.100.1"

Another part is if you want to access it via address you need to own a domain and point it to your public IP address and in case you got dynamic address from your ISP which is usually the case you would need to either set up DDNS also on the mentioned router or run it as a service on your machine or container etc...

For that I can recommend https://www.dynu.com/

Example of some config parts below.
![image](https://github.com/user-attachments/assets/87814253-d1be-4d79-9dc5-0caed6a1b954)

![image](https://github.com/user-attachments/assets/0ecb0cb0-e0ac-442c-b467-c3858a984318)

If you want help setting it up, be ready to pay for a beer before reaching out.
